Error installing myMPD

The install process fails because missing PCRE libraries. 

Also the installation process does not seem to be isolated in a transaction, so it leaves the system in a dirty state, users have to do some apt gymnastics to clean up.

Additionaly the "copy-only-blue-text" feature did not produce any meaningful content inside the blue text area, so I am posting everything.

<details>
  <summary>Logfile attached. Click to expand!</summary>

```
DietPi-Software
─────────────────────────────────────────────────────
 Step: Installing myMPD: fork of ympd with improved features

[  OK  ] DietPi-Software | Checking URL: https://github.com/jcorporation/myMPD/archive/master.tar.gz
[  OK  ] DietPi-Software | cd /tmp/DietPi-Software
[ INFO ] DietPi-Software | G_THREAD_START_0 | curl -sSfL https://github.com/jcorporation/myMPD/archive/master.tar.gz -o master.tar.gz
[ INFO ] DietPi-Software | APT install for: make gcc cmake perl libc6-dev libssl-dev libid3tag0-dev libflac-dev liblua5.3-dev pkg-config libpcre3-dev, please wait...
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package perl-modules-5.32.
(Reading database ... 44898 files and directories currently installed.)
Preparing to unpack .../00-perl-modules-5.32_5.32.1-4+deb11u2_all.deb ...
Unpacking perl-modules-5.32 (5.32.1-4+deb11u2) ...
Selecting previously unselected package libgdbm6:arm64.
Preparing to unpack .../01-libgdbm6_1.19-2_arm64.deb ...
Unpacking libgdbm6:arm64 (1.19-2) ...
Selecting previously unselected package libgdbm-compat4:arm64.
Preparing to unpack .../02-libgdbm-compat4_1.19-2_arm64.deb ...
Unpacking libgdbm-compat4:arm64 (1.19-2) ...
Selecting previously unselected package libperl5.32:arm64.
Preparing to unpack .../03-libperl5.32_5.32.1-4+deb11u2_arm64.deb ...
Unpacking libperl5.32:arm64 (5.32.1-4+deb11u2) ...
Selecting previously unselected package perl.
Preparing to unpack .../04-perl_5.32.1-4+deb11u2_arm64.deb ...
Unpacking perl (5.32.1-4+deb11u2) ...
Selecting previously unselected package binutils-common:arm64.
Preparing to unpack .../05-binutils-common_2.35.2-2_arm64.deb ...
Unpacking binutils-common:arm64 (2.35.2-2) ...
Selecting previously unselected package libbinutils:arm64.
Preparing to unpack .../06-libbinutils_2.35.2-2_arm64.deb ...
Unpacking libbinutils:arm64 (2.35.2-2) ...
Selecting previously unselected package libctf-nobfd0:arm64.
Preparing to unpack .../07-libctf-nobfd0_2.35.2-2_arm64.deb ...
Unpacking libctf-nobfd0:arm64 (2.35.2-2) ...
Selecting previously unselected package libctf0:arm64.
Preparing to unpack .../08-libctf0_2.35.2-2_arm64.deb ...
Unpacking libctf0:arm64 (2.35.2-2) ...
Selecting previously unselected package binutils-aarch64-linux-gnu.
Preparing to unpack .../09-binutils-aarch64-linux-gnu_2.35.2-2_arm64.deb ...
Unpacking binutils-aarch64-linux-gnu (2.35.2-2) ...
Selecting previously unselected package binutils.
Preparing to unpack .../10-binutils_2.35.2-2_arm64.deb ...
Unpacking binutils (2.35.2-2) ...
Selecting previously unselected package cmake-data.
Preparing to unpack .../11-cmake-data_3.18.4-2+deb11u1_all.deb ...
Unpacking cmake-data (3.18.4-2+deb11u1) ...
Selecting previously unselected package libarchive13:arm64.
Preparing to unpack .../12-libarchive13_3.4.3-2+b1_arm64.deb ...
Unpacking libarchive13:arm64 (3.4.3-2+b1) ...
Selecting previously unselected package libjsoncpp24:arm64.
Preparing to unpack .../13-libjsoncpp24_1.9.4-4_arm64.deb ...
Unpacking libjsoncpp24:arm64 (1.9.4-4) ...
Selecting previously unselected package librhash0:arm64.
Preparing to unpack .../14-librhash0_1.4.1-2_arm64.deb ...
Unpacking librhash0:arm64 (1.4.1-2) ...
Selecting previously unselected package cmake.
Preparing to unpack .../15-cmake_3.18.4-2+deb11u1_arm64.deb ...
Unpacking cmake (3.18.4-2+deb11u1) ...
Selecting previously unselected package libcc1-0:arm64.
Preparing to unpack .../16-libcc1-0_10.2.1-6_arm64.deb ...
Unpacking libcc1-0:arm64 (10.2.1-6) ...
Selecting previously unselected package libitm1:arm64.
Preparing to unpack .../17-libitm1_10.2.1-6_arm64.deb ...
Unpacking libitm1:arm64 (10.2.1-6) ...
Selecting previously unselected package libatomic1:arm64.
Preparing to unpack .../18-libatomic1_10.2.1-6_arm64.deb ...
Unpacking libatomic1:arm64 (10.2.1-6) ...
Selecting previously unselected package libasan6:arm64.
Preparing to unpack .../19-libasan6_10.2.1-6_arm64.deb ...
Unpacking libasan6:arm64 (10.2.1-6) ...
Selecting previously unselected package liblsan0:arm64.
Preparing to unpack .../20-liblsan0_10.2.1-6_arm64.deb ...
Unpacking liblsan0:arm64 (10.2.1-6) ...
Selecting previously unselected package libtsan0:arm64.
Preparing to unpack .../21-libtsan0_10.2.1-6_arm64.deb ...
Unpacking libtsan0:arm64 (10.2.1-6) ...
Selecting previously unselected package libubsan1:arm64.
Preparing to unpack .../22-libubsan1_10.2.1-6_arm64.deb ...
Unpacking libubsan1:arm64 (10.2.1-6) ...
Selecting previously unselected package libgcc-10-dev:arm64.
Preparing to unpack .../23-libgcc-10-dev_10.2.1-6_arm64.deb ...
Unpacking libgcc-10-dev:arm64 (10.2.1-6) ...
Selecting previously unselected package gcc-10.
Preparing to unpack .../24-gcc-10_10.2.1-6_arm64.deb ...
Unpacking gcc-10 (10.2.1-6) ...
Selecting previously unselected package gcc.
Preparing to unpack .../25-gcc_4%3a10.2.1-1_arm64.deb ...
Unpacking gcc (4:10.2.1-1) ...
Selecting previously unselected package libc-dev-bin.
Preparing to unpack .../26-libc-dev-bin_2.31-13+deb11u2_arm64.deb ...
Unpacking libc-dev-bin (2.31-13+deb11u2) ...
Selecting previously unselected package linux-libc-dev:arm64.
Preparing to unpack .../27-linux-libc-dev_21.08.6_arm64.deb ...
Unpacking linux-libc-dev:arm64 (21.08.6) ...
Selecting previously unselected package libcrypt-dev:arm64.
Preparing to unpack .../28-libcrypt-dev_1%3a4.4.18-4_arm64.deb ...
Unpacking libcrypt-dev:arm64 (1:4.4.18-4) ...
Selecting previously unselected package libtirpc-dev:arm64.
Preparing to unpack .../29-libtirpc-dev_1.3.1-1_arm64.deb ...
Unpacking libtirpc-dev:arm64 (1.3.1-1) ...
Selecting previously unselected package libnsl-dev:arm64.
Preparing to unpack .../30-libnsl-dev_1.3.0-2_arm64.deb ...
Unpacking libnsl-dev:arm64 (1.3.0-2) ...
Selecting previously unselected package libc6-dev:arm64.
Preparing to unpack .../31-libc6-dev_2.31-13+deb11u2_arm64.deb ...
Unpacking libc6-dev:arm64 (2.31-13+deb11u2) ...
Selecting previously unselected package libdpkg-perl.
Preparing to unpack .../32-libdpkg-perl_1.20.9_all.deb ...
Unpacking libdpkg-perl (1.20.9) ...
Selecting previously unselected package libogg-dev:arm64.
Preparing to unpack .../33-libogg-dev_1.3.4-0.1_arm64.deb ...
Unpacking libogg-dev:arm64 (1.3.4-0.1) ...
Selecting previously unselected package libflac-dev:arm64.
Preparing to unpack .../34-libflac-dev_1.3.3-2_arm64.deb ...
Unpacking libflac-dev:arm64 (1.3.3-2) ...
Selecting previously unselected package pkg-config.
Preparing to unpack .../35-pkg-config_0.29.2-1_arm64.deb ...
Unpacking pkg-config (0.29.2-1) ...
Selecting previously unselected package zlib1g-dev:arm64.
Preparing to unpack .../36-zlib1g-dev_1%3a1.2.11.dfsg-2_arm64.deb ...
Unpacking zlib1g-dev:arm64 (1:1.2.11.dfsg-2) ...
Selecting previously unselected package libid3tag0-dev.
Preparing to unpack .../37-libid3tag0-dev_0.15.1b-14_arm64.deb ...
Unpacking libid3tag0-dev (0.15.1b-14) ...
Selecting previously unselected package liblua5.3-0:arm64.
Preparing to unpack .../38-liblua5.3-0_5.3.3-1.1+b1_arm64.deb ...
Unpacking liblua5.3-0:arm64 (5.3.3-1.1+b1) ...
Selecting previously unselected package libncurses-dev:arm64.
Preparing to unpack .../39-libncurses-dev_6.2+20201114-2_arm64.deb ...
Unpacking libncurses-dev:arm64 (6.2+20201114-2) ...
Selecting previously unselected package libreadline-dev:arm64.
Preparing to unpack .../40-libreadline-dev_8.1-1_arm64.deb ...
Unpacking libreadline-dev:arm64 (8.1-1) ...
Selecting previously unselected package liblua5.3-dev:arm64.
Preparing to unpack .../41-liblua5.3-dev_5.3.3-1.1+b1_arm64.deb ...
Unpacking liblua5.3-dev:arm64 (5.3.3-1.1+b1) ...
Selecting previously unselected package libpcre16-3:arm64.
Preparing to unpack .../42-libpcre16-3_2%3a8.39-13_arm64.deb ...
Unpacking libpcre16-3:arm64 (2:8.39-13) ...
Selecting previously unselected package libpcre32-3:arm64.
Preparing to unpack .../43-libpcre32-3_2%3a8.39-13_arm64.deb ...
Unpacking libpcre32-3:arm64 (2:8.39-13) ...
Selecting previously unselected package libpcrecpp0v5:arm64.
Preparing to unpack .../44-libpcrecpp0v5_2%3a8.39-13_arm64.deb ...
Unpacking libpcrecpp0v5:arm64 (2:8.39-13) ...
Selecting previously unselected package libpcre3-dev:arm64.
Preparing to unpack .../45-libpcre3-dev_2%3a8.39-13_arm64.deb ...
Unpacking libpcre3-dev:arm64 (2:8.39-13) ...
Selecting previously unselected package libssl-dev:arm64.
Preparing to unpack .../46-libssl-dev_1.1.1k-1+deb11u1_arm64.deb ...
Unpacking libssl-dev:arm64 (1.1.1k-1+deb11u1) ...
Selecting previously unselected package make.
Preparing to unpack .../47-make_4.3-4.1_arm64.deb ...
Unpacking make (4.3-4.1) ...
Setting up libpcrecpp0v5:arm64 (2:8.39-13) ...
Setting up libpcre16-3:arm64 (2:8.39-13) ...
Setting up libarchive13:arm64 (3.4.3-2+b1) ...
Setting up perl-modules-5.32 (5.32.1-4+deb11u2) ...
Setting up binutils-common:arm64 (2.35.2-2) ...
Setting up linux-libc-dev:arm64 (21.08.6) ...
Setting up libctf-nobfd0:arm64 (2.35.2-2) ...
Setting up libogg-dev:arm64 (1.3.4-0.1) ...
Setting up libasan6:arm64 (10.2.1-6) ...
Setting up libtirpc-dev:arm64 (1.3.1-1) ...
Setting up make (4.3-4.1) ...
Setting up libssl-dev:arm64 (1.1.1k-1+deb11u1) ...
Setting up libpcre32-3:arm64 (2:8.39-13) ...
Setting up libatomic1:arm64 (10.2.1-6) ...
Setting up libjsoncpp24:arm64 (1.9.4-4) ...
Setting up libubsan1:arm64 (10.2.1-6) ...
Setting up libnsl-dev:arm64 (1.3.0-2) ...
Setting up librhash0:arm64 (1.4.1-2) ...
Setting up libcrypt-dev:arm64 (1:4.4.18-4) ...
Setting up liblua5.3-0:arm64 (5.3.3-1.1+b1) ...
Setting up cmake-data (3.18.4-2+deb11u1) ...
Setting up libbinutils:arm64 (2.35.2-2) ...
Setting up libc-dev-bin (2.31-13+deb11u2) ...
Setting up libcc1-0:arm64 (10.2.1-6) ...
Setting up liblsan0:arm64 (10.2.1-6) ...
Setting up libitm1:arm64 (10.2.1-6) ...
Setting up libgdbm6:arm64 (1.19-2) ...
Setting up libtsan0:arm64 (10.2.1-6) ...
Setting up libctf0:arm64 (2.35.2-2) ...
Setting up libflac-dev:arm64 (1.3.3-2) ...
Setting up libgcc-10-dev:arm64 (10.2.1-6) ...
Setting up binutils-aarch64-linux-gnu (2.35.2-2) ...
Setting up binutils (2.35.2-2) ...
Setting up libgdbm-compat4:arm64 (1.19-2) ...
Setting up libperl5.32:arm64 (5.32.1-4+deb11u2) ...
Setting up gcc-10 (10.2.1-6) ...
Setting up cmake (3.18.4-2+deb11u1) ...
Setting up libc6-dev:arm64 (2.31-13+deb11u2) ...
Setting up libncurses-dev:arm64 (6.2+20201114-2) ...
Setting up libpcre3-dev:arm64 (2:8.39-13) ...
Setting up libreadline-dev:arm64 (8.1-1) ...
Setting up gcc (4:10.2.1-1) ...
Setting up perl (5.32.1-4+deb11u2) ...
Setting up libdpkg-perl (1.20.9) ...
Setting up zlib1g-dev:arm64 (1:1.2.11.dfsg-2) ...
Setting up liblua5.3-dev:arm64 (5.3.3-1.1+b1) ...
Setting up pkg-config (0.29.2-1) ...
Setting up libid3tag0-dev (0.15.1b-14) ...
Processing triggers for libc-bin (2.31-13+deb11u2) ...
[  OK  ] DietPi-Software | APT install for: make gcc cmake perl libc6-dev libssl-dev libid3tag0-dev libflac-dev liblua5.3-dev pkg-config libpcre3-dev
[  OK  ] DietPi-Software | G_THREAD: All threads finished
[  OK  ] DietPi-Software | tar xf master.tar.gz
[  OK  ] DietPi-Software | rm master.tar.gz
[  OK  ] DietPi-Software | cd myMPD-master
[FAILED] DietPi-Software | ./build.sh releaseinstall

---------------------------------------------------------------------
- DietPi has encountered an error                                   -
- Please create a ticket: https://github.com/MichaIng/DietPi/issues -
- Copy and paste only the BLUE lines below into the ticket          -
---------------------------------------------------------------------
#### Details:
- Date           | Sun Dec 26 09:22:08 CET 2021
- DietPi version | v7.9.3 (MichaIng/master)
- Image creator  | DietPi Core Team
- Pre-image      | Armbian
- Hardware       | PINE A64 (aarch64) (ID=40)
- Kernel version | `Linux DietPi 5.10.60-sunxi64 #21.08.1 SMP Wed Aug 25 18:29:57 UTC 2021 aarch64 GNU/Linux`
- Distro         | bullseye (ID=6)
- Command        | `./build.sh releaseinstall`
- Exit code      | 1
- Software title | DietPi-Software
#### Steps to reproduce:
<!-- Explain how to reproduce the issue -->
1. ...
2. ...
#### Expected behaviour:
<!-- What SHOULD happen? -->
- ...
#### Actual behaviour:
<!-- What IS happening? -->
- ...
#### Extra details:
<!-- Please post any extra details that might help solve the issue -->
- ...
#### Additional logs:

Creating assets in release
Creating i18n json
WARN: jq not found - can not print translation statistics
Minifying javascript
Creating mympd.js
Minifying htdocs/sw.js
Minifying release/htdocs/js/mympd.js
Combining and compressing javascript
release/htdocs/js/combined.js:	 77.7% -- replaced with release/htdocs/js/combined.js.gz
release/htdocs/sw.min.js:	 59.4%
Minifying stylesheets
Minifying htdocs/css/mympd.css
Minifying htdocs/css/theme-light.css
Combining and compressing stylesheets
release/htdocs/css/combined.css:	 84.8% -- replaced with release/htdocs/css/combined.css.gz
Minifying and compressing html
Minifying htdocs/index.html
release/htdocs/index.html:	 87.3%
Creating other compressed assets
htdocs/mympd.webmanifest:	 61.2%
htdocs/assets/coverimage-booklet.svg:	 46.0%
htdocs/assets/coverimage-loading.svg:	 48.0%
htdocs/assets/coverimage-mympd.svg:	 51.5%
htdocs/assets/coverimage-notavailable.svg:	 47.9%
htdocs/assets/coverimage-stream.svg:	 47.0%
htdocs/assets/mympd-background-dark.svg:	 53.1%
htdocs/assets/mympd-background-light.svg:	 52.6%
Compiling myMPD
-- The C compiler identification is GNU 10.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
Compiler: GNU 10.2.1
BIN_PATH: /usr/bin
VARLIB_PATH: /var/lib/mympd
VARCACHE_PATH: /var/cache/mympd
Emedding assets in binary
ASSETS_PATH: /var/lib/mympd/empty
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.2")
CMake Error at /usr/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:165 (message):
  Could NOT find PCRE2 (missing: PCRE2_LIBRARY PCRE2_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:458 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindPCRE2.cmake:27 (find_package_handle_standard_args)
  CMakeLists.txt:92 (find_package)


-- Configuring incomplete, errors occurred!
See also "/tmp/DietPi-Software/myMPD-master/release/CMakeFiles/CMakeOutput.log".
See also "/tmp/DietPi-Software/myMPD-master/release/CMakeFiles/CMakeError.log".

---------------------------------------------------------------------
[FAILED] DietPi-Software | Unable to continue, DietPi-Software will now terminate.

```
</details>

--- a/CHANGELOG.txt
+++ b/CHANGELOG.txt
@@ -7,6 +7,7 @@ Stretch support:
 Changes:
 - DietPi-Software | Mono: "mono-complete" won't be installed anymore but only "mono-devel" instead. This skips the XSP4 webserver service, which listens on port 8084 by default where it conflicts with File Browser. This only affects new instals. You can manually apply this change via "apt-mark manual mono-devel && apt --autoremove purge mono-complete". Many thanks to @jaguar489 for reporting this issue: https://github.com/MichaIng/DietPi/issues/5093
 - DietPi-Software | FuguHub: The outdated official installer has been replaced with a fully automated own setup, removing the obsolete or even harmful interactive dialogues. On fresh installs, an admin account "dietpi" with global software password is created as well.
+- DietPi-Software | myMPD: Installation is now done via official APT repository, which means quicker install compared to compiling from source, fewer dependencies and easier updates via "apt upgrade": https://github.com/MichaIng/DietPi/issues/5115
 
 Fixes:
 - Raspberry Pi | Resolved an issue where unintentionally the turbo mode was enabled. This got backported to v7.9, to our existing RPi images and via live patch. Many thanks to @ayo-x and @whyisthisbroken for reporting this issue: https://github.com/MichaIng/DietPi/issues/5088
@@ -21,6 +22,7 @@ Fixes:
 - DietPi-Software | Mycroft AI: Resolved an issue on Bullseye (and above) systems where "mycroft-cli-client" command fails with a permissions issue, even as root user. Many thanks to @berndverhofstadt for reporting this issue: https://github.com/MichaIng/DietPi/issues/5100
 - DietPi-Software | Nukkit: Resolved an issue where the install failed due to a changed download URL. The Jenkins instance has moved to ci.opencollab.dev, where also the Geyser and Floodgate projects are hosted.
 - DietPi-Software | FuguHub: Resolved an issue where the uninstall failed as the process was not stopped as intended. Many thanks to @kd9352 for reporting this issue: https://github.com/MichaIng/DietPi/issues/5058
+- DietPi-Software | myMPD: Resolved an issue where the installation failed due to an updated dependency. Many thanks to @supersexy for reporting this issue: https://github.com/MichaIng/DietPi/issues/5115
 
 As always, many smaller code performance and stability improvements, visual and spelling fixes have been done, too much to list all of them here. Check out all code changes of this release on GitHub: https://github.com/MichaIng/DietPi/pull/XXXX
 

--- a/dietpi/dietpi-software
+++ b/dietpi/dietpi-software
@@ -5338,7 +5338,7 @@ _EOF_
 			fi
 
 			# APT key
-			G_EXEC curl -sSfL "https://download.opensuse.org/repositories/home:/jcorporation/$distro/Release.gpg" -o /etc/apt/trusted.gpg.d/dietpi-mympd.gpg
+			G_EXEC eval "curl -sSfL 'https://download.opensuse.org/repositories/home:/jcorporation/$distro/Release.key' | gpg --dearmor -o /etc/apt/trusted.gpg.d/dietpi-mympd.gpg --yes"
 
 			# APT list
 			G_EXEC eval "echo 'deb https://download.opensuse.org/repositories/home:/jcorporation/$distro/ /' > /etc/apt/sources.list.d/dietpi-mympd.list"

--- a/dietpi/dietpi-software
+++ b/dietpi/dietpi-software
@@ -8160,14 +8160,14 @@ _EOF_
 				fi
 
 				# Pre-v6.26 cleanup
-				[[ -f '/etc/openvpn/dh2048.pem' ]] && rm /etc/openvpn/dh2048.pem
+				[[ -f '/etc/openvpn/dh2048.pem' ]] && G_EXEC_NOEXIT=1 G_EXEC rm /etc/openvpn/dh2048.pem
 				dpkg-query -s easy-rsa &> /dev/null && G_AGP easy-rsa
 
 				# Download latest easy-rsa from GitHub
 				G_DIETPI-NOTIFY 2 'Downloading latest easy-rsa for certificate and key generation...'
 				local fallback_url='https://github.com/OpenVPN/easy-rsa/releases/download/v3.0.8/EasyRSA-3.0.8.tgz'
 				Download_Install "$(curl -sSfL 'https://api.github.com/repos/OpenVPN/easy-rsa/releases/latest' | mawk -F\" '/"browser_download_url": .*\/EasyRSA-[^"\/]*\.tgz"/{print $4}')"
-				[[ -d '/etc/openvpn/easy-rsa' ]] && rm -R /etc/openvpn/easy-rsa
+				[[ -d '/etc/openvpn/easy-rsa' ]] && G_EXEC rm -R /etc/openvpn/easy-rsa
 				G_EXEC mv EasyRSA-* /etc/openvpn/easy-rsa
 
 				# Cert and key generation via easy-rsa
@@ -8804,8 +8804,8 @@ _EOF_
 
 			else
 
-				# - Move data structure
-				[[ -e $target_data_dir ]] && rm -R $target_data_dir
+				# Move data structure
+				[[ -e $target_data_dir ]] && G_EXEC rm -R $target_data_dir
 				G_EXEC mv /var/www/pydio/data $target_data_dir
 
 			fi
@@ -12753,7 +12753,7 @@ _EOF_
 
 			Banner_Uninstalling
 			G_AGP xrdp xorgxrdp
-			[[ -f '/etc/apt/preferences.d/dietpi-xrdp' ]] && rm -v /etc/apt/preferences.d/dietpi-xrdp
+			[[ -f '/etc/apt/preferences.d/dietpi-xrdp' ]] && G_EXEC rm /etc/apt/preferences.d/dietpi-xrdp
 
 		fi
 
@@ -12762,10 +12762,10 @@ _EOF_
 
 			Banner_Uninstalling
 			G_AGP transmission-daemon
-			getent passwd debian-transmission > /dev/null && userdel debian-transmission
-			getent group debian-transmission > /dev/null && groupdel debian-transmission
-			[[ -f '/etc/systemd/system/transmission-daemon.service' ]] && rm /etc/systemd/system/transmission-daemon.service # pre v6.17
-			[[ -d '/etc/systemd/system/transmission-daemon.service.d' ]] && rm -R /etc/systemd/system/transmission-daemon.service.d # post v6.17
+			getent passwd debian-transmission > /dev/null && G_EXEC userdel debian-transmission
+			getent group debian-transmission > /dev/null && G_EXEC groupdel debian-transmission
+			[[ -f '/etc/systemd/system/transmission-daemon.service' ]] && G_EXEC rm /etc/systemd/system/transmission-daemon.service # pre v6.17
+			[[ -d '/etc/systemd/system/transmission-daemon.service.d' ]] && G_EXEC rm -R /etc/systemd/system/transmission-daemon.service.d # post v6.17
 
 		fi
 
@@ -12814,7 +12814,7 @@ _EOF_
 				rm /etc/apache2/conf-available/dietpi-dav_redirect.conf
 
 			fi
-			grep -q 'owncloud' /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf &> /dev/null && rm /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf
+			grep -q 'owncloud' /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf &> /dev/null && G_EXEC rm /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf
 
 		fi
 
@@ -12823,8 +12823,8 @@ _EOF_
 
 			Banner_Uninstalling
 			G_AGP coturn
-			[[ -f '/etc/turnserver.conf' ]] && rm /etc/turnserver.conf
-			[[ -d '/etc/systemd/system/coturn.service.d' ]] && rm -R /etc/systemd/system/coturn.service.d
+			[[ -f '/etc/turnserver.conf' ]] && G_EXEC rm /etc/turnserver.conf
+			[[ -d '/etc/systemd/system/coturn.service.d' ]] && G_EXEC rm -R /etc/systemd/system/coturn.service.d
 			systemctl start mariadb
 			systemctl start redis-server
 			ncc maintenance:mode --off
@@ -12877,7 +12877,7 @@ _EOF_
 				rm /etc/apache2/conf-available/dietpi-dav_redirect.conf
 
 			fi
-			grep -q 'nextcloud' /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf &> /dev/null && rm /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf
+			grep -q 'nextcloud' /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf &> /dev/null && G_EXEC rm /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf
 
 		fi
 
@@ -12885,7 +12885,7 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
-			[[ -d '/var/www/linuxdash' ]] && rm -R /var/www/linuxdash
+			[[ -d '/var/www/linuxdash' ]] && G_EXEC rm -R /var/www/linuxdash
 
 		fi
 
@@ -12893,14 +12893,14 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
-			[[ -d '/var/www/tasmoadmin' ]] && rm -R /var/www/tasmoadmin
+			[[ -d '/var/www/tasmoadmin' ]] && G_EXEC rm -R /var/www/tasmoadmin
 
 			# Remove webserver configs
 			command -v a2dissite > /dev/null && a2dissite dietpi-tasmoadmin
-			[[ -f '/etc/apache2/conf-available/dietpi-tasmoadmin.conf' ]] && rm /etc/apache2/conf-available/dietpi-tasmoadmin.conf
-			[[ -f '/etc/nginx/sites-dietpi/dietpi-tasmoadmin.conf' ]] && rm /etc/nginx/sites-dietpi/dietpi-tasmoadmin.conf
+			[[ -f '/etc/apache2/conf-available/dietpi-tasmoadmin.conf' ]] && G_EXEC rm /etc/apache2/conf-available/dietpi-tasmoadmin.conf
+			[[ -f '/etc/nginx/sites-dietpi/dietpi-tasmoadmin.conf' ]] && G_EXEC rm /etc/nginx/sites-dietpi/dietpi-tasmoadmin.conf
 			command -v lighty-disable-mod > /dev/null && lighty-disable-mod dietpi-tasmoadmin
-			[[ -f '/etc/lighttpd/conf-available/99-dietpi-tasmoadmin.conf' ]] && rm /etc/lighttpd/conf-available/99-dietpi-tasmoadmin.conf
+			[[ -f '/etc/lighttpd/conf-available/99-dietpi-tasmoadmin.conf' ]] && G_EXEC rm /etc/lighttpd/conf-available/99-dietpi-tasmoadmin.conf
 
 		fi
 
@@ -13004,7 +13004,7 @@ _EOF_
 				G_EXEC systemctl disable --now tor
 				G_EXEC rm /lib/systemd/system/tor.service
 			fi
-			[[ -d '/etc/systemd/system/tor.service.d' ]] && rm -R /etc/systemd/system/tor.service.d
+			[[ -d '/etc/systemd/system/tor.service.d' ]] && G_EXEC rm -R /etc/systemd/system/tor.service.d
 			G_AGP tor obfs4proxy
 		fi
 
@@ -13015,8 +13015,8 @@ _EOF_
 			systemctl start mariadb
 			mysqladmin -f drop phpbb drop phpbb3 # phpbb3: Pre-v6.33
 			mysql -e 'drop user phpbb@localhost; drop user phpbb3@localhost' # phpbb3: Pre-v6.33
-			[[ -d '/var/www/phpbb' ]] && rm -R /var/www/phpbb
-			[[ -d '/var/www/phpBB3' ]] && rm -R /var/www/phpBB3 # Pre-v6.33
+			[[ -d '/var/www/phpbb' ]] && G_EXEC rm -R /var/www/phpbb
+			[[ -d '/var/www/phpBB3' ]] && G_EXEC rm -R /var/www/phpBB3 # Pre-v6.33
 
 		fi
 
@@ -13074,7 +13074,8 @@ _EOF_
 			command -v mympd > /dev/null && G_EXEC rm "$(command -v mympd)"
 			command -v mympd-config > /dev/null && G_EXEC rm "$(command -v mympd-config)" # myMPD pre-v8.0.0
 			command -v mympd-script > /dev/null && G_EXEC rm "$(command -v mympd-script)"
-			G_EXEC rm -Rf /usr/share/doc/mympd /usr/share/man/man1/mympd* /etc/mympd.conf* # /etc: myMPD pre-v8.0.0
+			[[ -d '/usr/share/doc/mympd' ]] && G_EXEC rm -R /usr/share/doc/mympd
+			G_EXEC rm -f /usr/share/man/man1/mympd* /etc/mympd.conf* # /etc: myMPD pre-v8.0.0
 
 		fi
 
@@ -13225,7 +13226,7 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
-			[[ -d '/var/www/aria2' ]] && rm -R /var/www/aria2
+			[[ -d '/var/www/aria2' ]] && G_EXEC rm -R /var/www/aria2
 			if [[ -f '/etc/systemd/system/aria2.service' ]]
 			then
 				G_EXEC systemctl disable --now aria2
@@ -13444,8 +13445,8 @@ _EOF_
 			fi
 
 			G_AGP unbound
-			[[ -d '/etc/unbound' ]] && rm -R /etc/unbound
-			[[ -d '/var/cache/unbound' ]] && rm -R /var/cache/unbound
+			[[ -d '/etc/unbound' ]] && G_EXEC rm -R /etc/unbound
+			[[ -d '/var/cache/unbound' ]] && G_EXEC rm -R /var/cache/unbound
 		fi
 
 		software_id=143 # Koel
@@ -14414,10 +14415,10 @@ _EOF_
 
 			fi
 			# - Nginx
-			[[ -f '/etc/nginx/sites-dietpi/dietpi-dav_redirect.conf' ]] && rm -v /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf
-			[[ -f '/etc/nginx/sites-dietpi/dietpi-baikal.conf' ]] && rm -v /etc/nginx/sites-dietpi/dietpi-baikal.conf
+			[[ -f '/etc/nginx/sites-dietpi/dietpi-dav_redirect.conf' ]] && G_EXEC rm /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf
+			[[ -f '/etc/nginx/sites-dietpi/dietpi-baikal.conf' ]] && G_EXEC rm /etc/nginx/sites-dietpi/dietpi-baikal.conf
 
-			[[ -d '/var/www/baikal' ]] && rm -R /var/www/baikal
+			[[ -d '/var/www/baikal' ]] && G_EXEC rm -R /var/www/baikal
 
 			# Drop database
 			systemctl start mariadb

--- a/dietpi/dietpi-software
+++ b/dietpi/dietpi-software
@@ -5330,7 +5330,7 @@ _EOF_
 			then
 				distro+='10'
 
-			elif (( $G_DISTRO == 6 )) || [[ distro == 'Raspbian_' ]] # No Testing/Bullseye suite available for Raspbian yet
+			elif (( $G_DISTRO == 6 )) || [[ $distro == 'Raspbian_' ]] # No Testing/Bullseye suite available for Raspbian yet
 			then
 				distro+='11'
 			else

--- a/dietpi/dietpi-software
+++ b/dietpi/dietpi-software
@@ -13066,6 +13066,7 @@ _EOF_
 			getent passwd mympd > /dev/null && G_EXEC userdel mympd
 			getent group mympd > /dev/null && G_EXEC groupdel mympd
 			[[ -d '/var/lib/mympd' ]] && G_EXEC rm -R /var/lib/mympd
+			[[ -d '/var/cache/mympd' ]] && G_EXEC rm -R /var/cache/mympd
 			[[ -f '/etc/apt/sources.list.d/dietpi-mympd.list' ]] && G_EXEC rm /etc/apt/sources.list.d/dietpi-mympd.list
 			[[ -f '/etc/apt/trusted.gpg.d/dietpi-mympd.gpg' ]] && G_EXEC rm /etc/apt/trusted.gpg.d/dietpi-mympd.gpg
 			# Pre-v8.0

--- a/dietpi/dietpi-software
+++ b/dietpi/dietpi-software
@@ -5322,20 +5322,31 @@ _EOF_
 
 			Banner_Installing
 
-			# Install dependencies: https://github.com/jcorporation/myMPD/blob/master/build.sh#L615-L616
-			DEPS_LIST='make gcc cmake perl libc6-dev libssl-dev libid3tag0-dev libflac-dev liblua5.3-dev pkg-config libpcre3-dev'
+			# Distro: https://download.opensuse.org/repositories/home:/jcorporation/
+			local distro='Debian_'
+			(( $G_HW_MODEL < 10 )) && (( $G_RASPBIAN )) && distro='Raspbian_'
 
-			# Download source
-			Download_Install 'https://github.com/jcorporation/myMPD/archive/master.tar.gz'
+			if (( $G_DISTRO == 5 ))
+			then
+				distro+='10'
 
-			# Build and install myMPD
-			G_EXEC cd myMPD-master
-			G_EXEC ./build.sh releaseinstall
+			elif (( $G_DISTRO == 6 )) || [[ distro == 'Raspbian_' ]] # No Testing/Bullseye suite available for Raspbian yet
+			then
+				distro+='11'
+			else
+				distro+='Testing'
+			fi
+
+			# APT key
+			G_EXEC curl -sSfL "https://download.opensuse.org/repositories/home:/jcorporation/$distro/Release.gpg" -o /etc/apt/trusted.gpg.d/dietpi-mympd.gpg
+
+			# APT list
+			G_EXEC eval "echo 'deb https://download.opensuse.org/repositories/home:/jcorporation/$distro/ /' > /etc/apt/sources.list.d/dietpi-mympd.list"
+			G_AGUP
+
+			# APT package
+			G_AGI mympd
 			G_EXEC systemctl stop mympd
-
-			# Cleanup
-			G_EXEC cd /tmp/$G_PROGRAM_NAME
-			G_EXEC_NOHALT=1 G_EXEC rm -R myMPD-master
 
 			# User: Switch to primary group "dietpi" to allow media access and create media files as "dietpi" group: https://github.com/MichaIng/DietPi/issues/3382
 			G_EXEC usermod -g dietpi mympd
@@ -5357,10 +5368,10 @@ _EOF_
 				G_EXEC chown -R mympd:dietpi /var/lib/mympd/state
 			fi
 
-			# myMPD pre-v8.0.0 migration: Remove obsolete file
+			# myMPD pre-v8.0.0 migration: Remove obsolete files
 			[[ -f '/etc/mympd.conf' ]] && G_EXEC_NOEXIT=1 G_EXEC rm /etc/mympd.conf
 			[[ -f '/etc/mympd.conf.dist' ]] && G_EXEC_NOEXIT=1 G_EXEC rm /etc/mympd.conf.dist
-			command -v mympd-config > /dev/null && G_EXEC rm "$(command -v mympd-config)"
+			command -v mympd-config > /dev/null && G_EXEC_NOEXIT=1 G_EXEC rm "$(command -v mympd-config)"
 
 		fi
 
@@ -13044,6 +13055,9 @@ _EOF_
 
 		software_id=148 # myMPD
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
+
+			Banner_Uninstalling
+			G_AGP mympd
 
 			if [[ -f '/lib/systemd/system/mympd.service' ]]
 			then

--- a/dietpi/dietpi-software
+++ b/dietpi/dietpi-software
@@ -5349,8 +5349,8 @@ _EOF_
 			G_EXEC systemctl stop mympd
 
 			# User: Switch to primary group "dietpi" to allow media access and create media files as "dietpi" group: https://github.com/MichaIng/DietPi/issues/3382
+			# - Leave "mympd" group in place, as it is recreated on package upgrades
 			G_EXEC usermod -g dietpi mympd
-			getent group mympd > /dev/null && G_EXEC_NOEXIT=1 G_EXEC groupdel mympd
 
 			# Config: Create on fresh install
 			# - On reinstall /var/lib/mympd exists, so use its existence as reinstall flag
@@ -5365,7 +5365,7 @@ _EOF_
 				echo '/mnt/dietpi_userdata/Music' > /var/lib/mympd/state/playlist_directory
 				echo '/mnt/dietpi_userdata/Music' > /var/lib/mympd/state/music_directory
 				G_EXEC chmod 0600 /var/lib/mympd/state/*
-				G_EXEC chown -R mympd:dietpi /var/lib/mympd/state
+				G_EXEC chown -R mympd:root /var/lib/mympd/state
 			fi
 
 			# myMPD pre-v8.0.0 migration: Remove obsolete files
@@ -13023,11 +13023,10 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
-			if [[ -f '/etc/systemd/system/webmin.service' ]]; then
-
+			if [[ -f '/etc/systemd/system/webmin.service' ]]
+			then
 				G_EXEC systemctl disable --now webmin
 				G_EXEC rm /etc/systemd/system/webmin.service
-
 			fi
 			[[ -d '/etc/systemd/system/webmin.service.d' ]] && G_EXEC rm -R /etc/systemd/system/webmin.service.d
 			G_AGP webmin
@@ -13040,11 +13039,10 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
-			if [[ -f '/etc/systemd/system/ympd.service' ]]; then
-
+			if [[ -f '/etc/systemd/system/ympd.service' ]]
+			then
 				G_EXEC systemctl disable --now ympd
 				G_EXEC rm /etc/systemd/system/ympd.service
-
 			fi
 			[[ -d '/etc/systemd/system/ympd.service.d' ]] && G_EXEC rm -R /etc/systemd/system/ympd.service.d
 			getent passwd ympd > /dev/null && G_EXEC userdel ympd
@@ -13057,8 +13055,6 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
-			G_AGP mympd
-
 			if [[ -f '/lib/systemd/system/mympd.service' ]]
 			then
 				G_EXEC systemctl unmask mympd
@@ -13066,14 +13062,15 @@ _EOF_
 				G_EXEC rm /lib/systemd/system/mympd.service
 			fi
 			[[ -d '/etc/systemd/system/mympd.service.d' ]] && G_EXEC rm -R /etc/systemd/system/mympd.service.d
-
+			G_AGP mympd
 			getent passwd mympd > /dev/null && G_EXEC userdel mympd
-			getent group mympd > /dev/null && G_EXEC groupdel mympd # pre-v6.29
-
+			getent group mympd > /dev/null && G_EXEC groupdel mympd
+			[[ -d '/var/lib/mympd' ]] && G_EXEC rm -R /var/lib/mympd
+			# Pre-v8.0
 			command -v mympd > /dev/null && G_EXEC rm "$(command -v mympd)"
 			command -v mympd-config > /dev/null && G_EXEC rm "$(command -v mympd-config)" # myMPD pre-v8.0.0
 			command -v mympd-script > /dev/null && G_EXEC rm "$(command -v mympd-script)"
-			G_EXEC rm -Rf /{usr/share/doc,var/lib}/mympd /usr/share/man/man1/mympd.* /etc/mympd.conf* # /etc: myMPD pre-v8.0.0
+			G_EXEC rm -Rf /usr/share/doc/mympd /usr/share/man/man1/mympd* /etc/mympd.conf* # /etc: myMPD pre-v8.0.0
 
 		fi
 
@@ -13081,18 +13078,17 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
-			if [[ -f '/lib/systemd/system/mpd.service' ]]; then
-
+			if [[ -f '/lib/systemd/system/mpd.service' ]]
+			then
 				G_EXEC systemctl unmask mpd
 				G_EXEC systemctl disable --now mpd
 				G_EXEC rm /lib/systemd/system/mpd.service
-
 			fi
 			[[ -d '/etc/systemd/system/mpd.service.d' ]] && G_EXEC rm -R /etc/systemd/system/mpd.service.d
 			[[ -f '/lib/systemd/system/mpd.socket' ]] && G_EXEC rm /lib/systemd/system/mpd.socket
+			G_AGP mpd libmpdclient2
 			getent passwd mpd > /dev/null && G_EXEC userdel mpd
 			getent group mpd > /dev/null && G_EXEC groupdel mpd # pre-v6.29
-			G_AGP mpd libmpdclient2
 			[[ -d '/var/log/mpd' ]] && G_EXEC rm -R /var/log/mpd
 			[[ -d '/mnt/dietpi_userdata/.mpd_cache' ]] && G_EXEC rm -R /mnt/dietpi_userdata/.mpd_cache
 			[[ -f '/etc/mpd.conf' ]] && G_EXEC rm /etc/mpd.conf
@@ -13107,11 +13103,10 @@ _EOF_
 			Banner_Uninstalling
 
 			# Service
-			if [[ -f '/etc/systemd/system/node-red.service' ]]; then
-
+			if [[ -f '/etc/systemd/system/node-red.service' ]]
+			then
 				G_EXEC systemctl disable --now node-red
 				G_EXEC rm /etc/systemd/system/node-red.service
-
 			fi
 			[[ -d '/etc/systemd/system/node-red.service.d' ]] && G_EXEC rm -R /etc/systemd/system/node-red.service.d
 
@@ -13146,11 +13141,10 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
-			if [[ -f '/etc/systemd/system/mosquitto.service' ]]; then
-
+			if [[ -f '/etc/systemd/system/mosquitto.service' ]]
+			then
 				G_EXEC systemctl disable --now mosquitto
 				G_EXEC rm /etc/systemd/system/mosquitto.service
-
 			fi
 			[[ -d '/etc/systemd/system/mosquitto.service.d' ]] && G_EXEC rm -R /etc/systemd/system/mosquitto.service.d
 			G_AGP mosquitto

--- a/dietpi/dietpi-software
+++ b/dietpi/dietpi-software
@@ -13066,6 +13066,8 @@ _EOF_
 			getent passwd mympd > /dev/null && G_EXEC userdel mympd
 			getent group mympd > /dev/null && G_EXEC groupdel mympd
 			[[ -d '/var/lib/mympd' ]] && G_EXEC rm -R /var/lib/mympd
+			[[ -f '/etc/apt/sources.list.d/dietpi-mympd.list' ]] && G_EXEC rm /etc/apt/sources.list.d/dietpi-mympd.list
+			[[ -f '/etc/apt/trusted.gpg.d/dietpi-mympd.gpg' ]] && G_EXEC rm /etc/apt/trusted.gpg.d/dietpi-mympd.gpg
 			# Pre-v8.0
 			command -v mympd > /dev/null && G_EXEC rm "$(command -v mympd)"
 			command -v mympd-config > /dev/null && G_EXEC rm "$(command -v mympd-config)" # myMPD pre-v8.0.0

--- a/dietpi/dietpi-software
+++ b/dietpi/dietpi-software
@@ -2760,6 +2760,7 @@ _EOF_
 
 			Banner_Installing
 			G_AGI avahi-daemon
+			G_EXEC systemctl stop avahi-daemon
 
 		fi
 

